nibabel/arrayproxy.py

```diff
@@ -64,17 +66,32 @@ class ArrayProxy(object):
     """
     # Assume Fortran array memory layout
     order = 'F'
+    _header = None
 
     @kw_only_meth(2)
-    def __init__(self, file_like, header, mmap=True):
+    def __init__(self, file_like, spec, mmap=True):
         """ Initialize array proxy instance
 
         Parameters
         ----------
         file_like : object
             File-like object or filename. If file-like object, should implement
             at least ``read`` and ``seek``.
-        header : object
+        spec : object or tuple
+            Tuple must have length 2-5, with the following fields:
```

Comment:
Used a period here. The rest seemed to be more fragments than sentences, but I can add periods there, if they're preferred, stylistically.